### DISCUSSION TITLE
Multiple SANS?

### DISCUSSION THREAD

**Discussion by cmblackburn** [Venafi/venafips#224] [General]
Does anyone know how to set multiple SAN values for IP and DNS names with the set-tppattribute?

**Comment by gdbarron** [comment]
This was answered in #107.  Multiple sans would just be additional key/value pairs, eg.
`Set-VdcAttribute -Path '\VED\Policy\Certificates\me.x.com' -Attribute @{'X509 SubjectAltName IPAddress'='1.2.3.4'; 'X509 SubjectAltName DNS'='me.x.com'}`
